grafana-plugin/src/containers/TeamsList/TeamsList.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { useCallback, useState } from 'react';
+
+import { Badge, Button, Field, HorizontalGroup, Modal, RadioButtonGroup, Tooltip, VerticalGroup } from '@grafana/ui';
+import { observer } from 'mobx-react';
+
+import Avatar from 'components/Avatar/Avatar';
+import GTable from 'components/GTable/GTable';
+import Text from 'components/Text/Text';
+import { WithPermissionControlTooltip } from 'containers/WithPermissionControl/WithPermissionControlTooltip';
+import { GrafanaTeam } from 'models/grafana_team/grafana_team.types';
+import { useStore } from 'state/useStore';
+import { UserActions } from 'utils/authorization';
+
+const TeamsList = observer(() => {
+  const store = useStore();
+  const [teamIdToShowModal, setTeamIdToShowModal] = useState<GrafanaTeam['id']>();
+  const { userStore } = store;
+
+  const isTeamDefault = (record: GrafanaTeam) => {
+    return (
+      (record.id === 'null' && store.userStore.currentUser?.current_team === null) ||
+      record.id === store.userStore.currentUser?.current_team
+    );
+  };
+
+  const renderTeam = (record: GrafanaTeam) => {
+    return (
+      <>
+        {record.id === 'null' ? (
+          <Badge text={record.name} color={'blue'} tooltip={'Resource is not assigned to any team (ex General team)'} />
```

Comment:
similar comment here regarding literal `'null'` string and "General" team name

Reply:
answered above